### DISCUSSION TITLE
License Doesn't work on Edge

### DISCUSSION THREAD

**Discussion by cramee0907** [aclap-dev/video-downloadhelper#363] [License & Purchase requests]
### What's your issue?

I bought a license on edge and registered a license, but when I try to download it, it says try after 120 minutes. Is there a fix?

### The 10 first digits (only 10) of your license number or email address

dldbtjq87@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
What do you see in the license section of the settings page? https://github.com/aclap-dev/video-downloadhelper/wiki#register-your-premium

**Reply by cramee0907** [reply]
my bad. Problem solved
